Open Judge System/MongoDBPoC/Program.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace MongoDBPoC
+{
+    using System;
+    using System.Collections.Generic;
+
+    using MongoDB.Bson;
+    using MongoDB.Driver;
+
+    public class Program
+    {
+        static void Main()
+        {
+            const string ConnectionString = "mongodb://localhost:27017";
+            var client = new MongoClient(ConnectionString);
+            IMongoDatabase db = client.GetDatabase("OJS");
+            var collection = db.GetCollection<SubmissionForProcessing>("submissionsForProcessing");
+
+            InsertSubmissions(collection);
+
+            var filter = new FilterDefinitionBuilder<SubmissionForProcessing>().Where(s => s.Processed);
+            collection.Find(filter).ToList()
+                .ForEach(s => Console.WriteLine(
+                             $"ID: {s.SubmissionId}; Processed: {s.Processed}; Processing: {s.Processing}"));
+
+        }
+
+        private static void InsertSubmissions(IMongoCollection<SubmissionForProcessing> collection)
```

Comment:
IMO leave a TODO to erase the PoC after it is implemented. You may add an issues that lists all PoC that are no longer needed -> to be erased from project.